src/js/lib/jquery.kazoosdk.js

```diff
@@ -165,7 +165,8 @@
 				'getStatus': { verb: 'GET', url: 'accounts/{accountId}/devices/status' },
 				'quickcall': { verb: 'GET', url: 'accounts/{accountId}/devices/{deviceId}/quickcall/{number}' },
 				'restart': { verb: 'POST', url: 'accounts/{accountId}/devices/{deviceId}/sync' },
-				'updatePresence': { verb: 'POST', url: 'accounts/{accountId}/device/{deviceId}/presence' }
+				'updatePresence': { verb: 'POST', url: 'accounts/{accountId}/device/{deviceId}/presence' }.
+				'getDevice': { verb: 'GET', url: 'accounts/{accountId}/users/{userId}/devices' }
```

Comment:
This endpoint list devices for a given user, please rename the resource accordingly: `listDevicesByUser`.